### PR TITLE
feat: document contribution

### PR DESCRIPTION
**Pull request by leite08**
Ref. metriport/metriport-internal#228

### Dependencies

- Upstream: none
- Downstream: FHIR server infra TBD

### Description

- add document contribution to CW cert-runner
- add `getPatient()` to commonwell-sdk
  - did this so we could query Orgs to update their endpoints - turns out they don't send the FHIR endpoint data with the Org when queried, so I gave up of the idea of updating the org based on the .env file
- still trying to get Commonwell to connect to our FHIR server when we execute this, so we might need to update the code at some point (new PR)

### Release Plan

- nothing special, no automated release, anytime

### CODE REVIEW

File: packages/packages/commonwell-cert-runner/src/shared-person.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      patientData.details.birthDate,
+      patientData.details.gender.code,
+      patientData.details.address[0].zip
+    );
+    console.log(respPatient);
+  
+    let personId: string | undefined = undefined;
+    let patientId: string | undefined = undefined;
+  
+    // IF THERE'S A PATIENT, USE IT IT
```
Comment (Goncharo) [inline packages/packages/commonwell-cert-runner/src/shared-person.ts:23]: typo :`IT IT`

**Review by Goncharo — CHANGES_REQUESTED** [review]
minor comments

File: packages/packages/commonwell-cert-runner/commonwell_9DkXDqjcTO92lFj5uaxfr.file.response [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
+  "id": "urn:uuid:8ce03a6a-a159-49e3-8d8b-04d19e0608ff",
+  "content": {
+    "resourceType": "DocumentReference",
```
[the remark this answers is not in the document]
Comment (leite08) [reply packages/packages/commonwell-cert-runner/commonwell_9DkXDqjcTO92lFj5uaxfr.file.response:4]: no, my bad, it shouldn't have been pushed to remote... will remove

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,3 +1,6 @@
 {
-    "dotenv.enableAutocloaking": false
+    "dotenv.enableAutocloaking": false,
+    "workbench.colorCustomizations": {
+      "statusBar.background": "#007acc"
```
[the remark this answers is not in the document]
Comment (leite08) [reply .vscode/settings.json:4]: This is the default color... had to add it to the file bc VSCode on MacOS updates all windows when one of them has a custom color - as we have with internal.

I can change this one to the purple, but then I'd suggest changing the internal one back to the original color or something different, so its super clear which window we're using.

[the remark this answers is not in the document]
Comment (leite08) [reply .vscode/settings.json:4]: We discussed to leave as is for now, our purple doesn't have enough contrast against foreground black or white on that size (small status bar and text size).

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**